<!-- This is for calibrated tip -->
<arg name = "offset_x"    default = "-0.01165925"/>
<arg name = "offset_y"    default = "0.03700886"/>
<arg name = "offset_z"    default = "-0.05041287"/>
<!-- This is for the touch point (e.g., rcm point) -->
<arg name = "touch_point_x"    default = "0.13968086"/>
<arg name = "touch_point_y"    default = "-0.12883264"/>
<arg name = "touch_point_z"    default = "1.26896223"/>
